utils/R/kill.tunnel.R

```diff
@@ -3,12 +3,19 @@
 ##' @title kill.tunnel
 ##' @description kill tunnel to remote machine
 ##' @author Rob Kooper
-kill.tunnel <- function() {
-    if (exists("settings") && !is.null(settings$host$tunnel)) {
+kill.tunnel <- function(settings,exe=TRUE,data=TRUE) {
```

Comment:
being picky but we should make sure the whole file is either 2 or 4 spaces for a tab, not a mixture.